pkg/util/util.go

```diff
@@ -143,14 +143,14 @@ func ConvertLabelsStringToMap(labels string) (map[string]string, error) {
 	return labelsMap, nil
 }
 
+// GetRegionFromZone return the corresponding region name based on a zone
+// Example "us-central1-a" return "us-central1"
 func GetRegionFromZone(location string) (string, error) {
 	tokens := strings.Split(location, "-")
 	if len(tokens) != 3 {
-		return "", fmt.Errorf("Failed to parse location %v", location)
+		return "", fmt.Errorf("failed to parse location %v", location)
 	}
-
-	tokens = tokens[:2]
-	return strings.Join(tokens, "-"), nil
+	return strings.Join(tokens[0:len(tokens)-1], "-"), nil
```

Comment:
is it needed here to do `len(tokens)-1` when we already know that the length has to be 3?

Reply:
i don't i think its needed ,probably this would make it better `strings.Join(token[0:2],"-")` ?